cesium_app/util.py

```diff
@@ -66,6 +66,10 @@ def check_model_param_types(model_type, model_params, all_as_lists=False):
             if p["name"] == k:
                 param_entry = p
                 break
+        try:
```

Comment:
This could be combined as `for: ...; else: raise ...` instead of checking afterward if `param_entry` was defined.